docs/DeveloperGuide.md

```diff
@@ -255,13 +245,13 @@ shown in the activity diagram below.
         object. There is a need to create additional methods to update the `CompanyDetailPanel` when there is any changes
         made to the `Company` object.
 
-### 4.3 Find command
+### Find Feature
 
-#### 4.3.1 Implementation
+#### Implementation
```

Comment:
wait are we all removing all these headers, cut right now like this it will appear in the TOC?

Reply:
it originally already appears in the TOC

Reply:
Yeap, we planning to remove from the TOC? I mean either way is fine by me, but if we intend to remove from TOC then these needs to change